distribution/lib/Standard/Base/0.0.0-dev/src/Enso_Cloud/Enso_Secret.enso

```diff
@@ -44,17 +44,19 @@ type Enso_Secret
         Context.Output.if_enabled disabled_message="Creating a secret is forbidden as the Output context is disabled." panic=False <|
             if name.starts_with "connection-" then Error.throw (Illegal_Argument.Error "Secret name cannot start with 'connection-'") else
                 parent_dir = parent.if_nothing Enso_File.current_working_directory
-                # TODO verify if root needs special handling, maybe file a bug
-                parent_id_pair = if parent_dir.is_current_user_root then [] else
-                    parent_dir_asset = Existing_Enso_Asset.get_asset_reference_for parent_dir
-                    [["parentDirectoryId", parent_dir_asset.id]]
-                body = JS_Object.from_pairs [["name", name], ["value", value]]+parent_id_pair
-                handle_already_exists _ =
-                    message = "A secret with the name " + name.pretty + " already exists inside of directory " + parent_dir.name + "."
-                    Error.throw (Illegal_Argument.Error message)
-                error_handlers = Map.from_vector [["resource_already_exists", handle_already_exists]]
-                id = Utils.http_request_as_json HTTP_Method.Post Utils.secrets_api body error_handlers=error_handlers
-                Enso_Secret.Value name id
+                path = if name.contains "/" then Error.throw (Illegal_Argument.Error "Secret name cannot contain `/`.") else parent_dir.enso_path.resolve name
+                path.if_not_error <|
+                    # TODO verify if root needs special handling, maybe file a bug
```

Comment:
Did we do this TODO?

Reply:
No, thanks for bringing it up - I meant to do it, I think on Tuesday, but kept forgetting... Actually, originally I meant to do it weeks ago 🤦 

I will try to check it before merging this PR. I'll check it once I'm back in office (currently on train with weak internet and I need to download an IDE build to get re-authenticated on my travel laptop as my creds expired).

Reply:
Thanks for the reminder - after all I have removed the TODO and the workaround as after checking it turns out it's no longer needed.